comp110-worksheet-7/DirectoryUtils.cs

```diff
@@ -1,14 +1,29 @@
 ﻿using System;
 using System.Collections.Generic;
 using System.IO;
-using System.Linq;
-using System.Text;
-using System.Threading.Tasks;
 
 namespace comp110_worksheet_7
 {
 	public static class DirectoryUtils
 	{
+        public static int GoDeeper(string[] CurrentDirectory, int CurrentDepth, int MostDepth)
```

Comment:
A more descriptive name for this function would be useful. Also since it seems to be a helper function for `GetDepth`, moving it nearer there in the file would be helpful.